tutorials/cpi-sentiment-analysis/cpi-sentiment-analysis.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Set up SAP Hybris Marketing Cloud integration with Twitter  
-description: The SAP Cloud Platform Integration package implements an integration of social media data into SAP Hybris Marketing Cloud. It allows you to load and analyze social media data from Twitter into your SAP Hybris Marketing Cloud system.
+title: How to setup SAP Cloud Platform Integration for Twitter using the SAP Hybris Marketing Cloud System
+description: The SAP Cloud Integration package implements an integration of Social Media Data into SAP Hybris Marketing Cloud. It allows you to load and analyze Social Media data from Twitter into your SAP Hybris Marketing Cloud system.
```

Comment:
Again, consider shortening the official product names, to make this fit in the catalog.

Reply:
Not sure we have a shortening name for SAP Hybris Marketing cloud system

Reply:
OK.  So you have a description which doesn't fit in the tile.  Here is the tile as it appears in QA:  

![screen shot 2017-12-18 at 2 23 18 pm](https://user-images.githubusercontent.com/3226389/34123949-30338e50-e3ff-11e7-85fe-25f84f9fcd17.png)

Notice that the long name takes up 1/2 of the description (because the title is long, too).  The word "cloud" is cut off, as it the entire last sentence.